lib/storage/src/content_manager/consensus_state.rs

```diff
@@ -77,6 +81,13 @@ impl<C: CollectionContainer> ConsensusState<C> {
             on_consensus_op_apply: Default::default(),
             propose_sender,
             first_voter: Default::default(),
+            consensus_thread_status: RwLock::new(ConsensusThreadStatus::Working),
```

Comment:
So right now there are no places changing `consensus_thread_status` from error to working? 
Just want to make sure I get this :)

Reply:
Yes, if it fails - it stays in failed state until restart.

Reply:
The idea is to allow search and update operations on existing collections regardless of consensus errors. Retrying errors doesn't seem to make sense as they are mostly environment related and therefore unrecoverable from app point of view

Reply:
Thanks, it makes sense :+1: